plugins/common/functions

```diff
@@ -316,12 +316,20 @@ release_and_deploy() {
       IMAGE_SOURCE_TYPE="dockerfile"
     fi
 
+    DOKKU_APP_SKIP_DEPLOY="$(dokku config:get $APP DOKKU_APP_SKIP_DEPLOY || true)"
```

Comment:
We'll probably want to support global with an override. See this as an example: https://github.com/dokku/dokku/blob/v0.4.6/dokku#L96-L102